### PR TITLE
[TEST] Onboarding flow

### PR DESCRIPTION
**Pull request by aristidesstaffieri**
This adds unit tests to the onboarding flow, covering the following cases -

Create new account
Import account
Create new account with incorrect mnemonic confirmation
Import account and incorrectly confirm password

### CODE REVIEW

File: extension/tests/onboarding.test.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+test("Welcome page loads", async ({ page }) => {
+  await expect(
+    page.getByText("Welcome! Is this your first time using Freighter?"),
+  ).toBeVisible();
+  await expect(page.getByText("I’m going to need a seed phrase")).toBeVisible();
+  await expect(page.getByText("I’ve done this before")).toBeVisible();
+});
+
+test.only("Create new wallet", async ({ page }) => {
```
Comment (piyalbasu) [inline extension/tests/onboarding.test.ts:16]: let's omit `.only`

File: extension/tests/test-fixtures.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { test as base, chromium, BrowserContext } from "@playwright/test";
+import path from "path";
+
+export const test = base.extend<{
+  context: BrowserContext;
+  extensionId: string;
+}>({
+  context: async ({}, use) => {
+    const pathToExtension = path.join(__dirname, "../build");
+    console.log(pathToExtension);
```
Comment (piyalbasu) [inline extension/tests/test-fixtures.ts:10]: console.log

**Comment by piyalbasu** [comment]
This looks great! Would you mind adding screenshot testing, as well? https://playwright.dev/docs/test-snapshots

I'll be really interested to see how long this takes to run in CI with snapshots

File: extension/tests/onboarding.test.ts [hunk 3]
```diff
@@ -0,0 +1,144 @@
+import { shuffle } from "lodash";
```
Comment (piyalbasu) [inline extension/tests/onboarding.test.ts:1]: do you mind changing this dir name to `e2eTests` so we can differentiate between these and unit tests?

File: extension/tests/onboarding.test.ts (hunk 1 above)
Comment (aristidesstaffieri) [reply extension/tests/onboarding.test.ts:16]: ah whoops, thanks.

File: extension/tests/onboarding.test.ts (hunk 3 above)
Comment (aristidesstaffieri) [reply extension/tests/onboarding.test.ts:1]: yup can do

File: extension/tests/onboarding.test.ts (hunk 1 above)
Comment (aristidesstaffieri) [reply extension/tests/onboarding.test.ts]: removed in e99194053f8fb93000911ce5cb43589f3ef43894

File: extension/tests/test-fixtures.ts (hunk 2 above)
Comment (aristidesstaffieri) [reply extension/tests/test-fixtures.ts]: removed in e99194053f8fb93000911ce5cb43589f3ef43894

File: extension/tests/onboarding.test.ts (hunk 3 above)
Comment (aristidesstaffieri) [reply extension/tests/onboarding.test.ts]: renamed in e99194053f8fb93000911ce5cb43589f3ef43894

**Review by piyalbasu — APPROVED** [review]

**Pull request merged**